docs/en/operations/utils/index.md

```diff
@@ -1,7 +1,7 @@
 # ClickHouse Utility
 
-* [clickhouse-local](clickhouse-local.md) — Allows running SQL queries on data without stopping the ClickHouse server, similar to how `awk` does this.
-* [clickhouse-copier](clickhouse-copier.md) — Copies (and reshards) data from one cluster to another cluster.
-
+- [clickhouse-local](clickhouse-local.md) — Allows running SQL queries on data without stopping the ClickHouse server, similar to how `awk` does this.
+- [clickhouse-copier](clickhouse-copier.md) — Copies (and reshards) data from one cluster to another cluster.
+- [clickhouse-benchmark](clickhouse-benchmark.md) — Loads server with the custom queries and settings.
```

Comment:
Is it OK that there is a warning in docs check?
`WARNING:mkdocs.structure.pages:Documentation file 'operations/utils/index.md' contains a link to 'operations/utils/clickhouse-benchmark.md' which is not found in the documentation files.
`
[link](https://clickhouse-test-reports.s3.yandex.net/7947/e296ec56b5069e2406935117f252a01d6057c462/docs_check/docs_output.txt
)

Reply:
I'll fix it before merge, thank you :-).